packages/lib/config/networks/index.ts

```diff
@@ -12,7 +13,7 @@ import sepolia from './sepolia'
 import mode from './mode'
 import fraxtal from './fraxtal'
 import sonic from './sonic'
-import { NetworkConfig } from '../config.types'
+import hyperevm from './hyperevm'
```

Comment:
why not import this from the sdk?

Reply:
what do you mean? this are the frontend configurations (that have things like LBP conf, etc.) Are you referring to the viem conf? (in that case, it is not exported from the SDK)

Reply:
ya just tested you can import `CHAINS` and `ChainId` from sdk and then do like 

```
const networkConfig: NetworkConfig = {
  chainId: CHAINS[ChainId.HYPER_EVM].id,
  ...
```

Reply:
just do `chainId: ChainId.HYPER_EVM` here

Reply:
The SDK configuration is not working (type problems derived from different viem versions used) so I will leave the custom HyperEVM config in our app for now.